Terraform GitHub Actions updates (#988)
hashicorp/terraform-website: content/source/docs/github-actions/common-actions/index.html.md

---
layout: "github-actions"

> page_title: "Workflows - Terraform GitHub Actions"

Reviewer: ```suggestion
page_title: "Common Tasks - Terraform GitHub Actions"
```

sidebar_current: "docs-github-actions-common-actions"
---

> # Configuration

Reviewer: ```suggestion
# Common Tasks
```

Terraform GitHub Actions are designed to be configurable for most workflows.

> See the links below for examples of common workflows.

Reviewer: ```suggestion
However, some Terraform features require slightly different configuration when used in GitHub Actions. See the links below for examples of how to use various Terraform features within GitHub Actions.
```
